### DISCUSSION TITLE
Postal in Portainer

### DISCUSSION THREAD

**Discussion by FlorinBuffet** [postalserver/postal#3265] [Help with using Postal]
Hi there
Is there a tutorial on how to install Postal in Portainer?
Best Regards
Florin

**Comment by Talha5** [comment]
I believe setting it up on Docker isn’t too challenging. However, I couldn’t find any documentation for Portainer.

**Reply by FlorinBuffet** [reply]
The default installation tutorial even is for docker. But it also has some commands in it like `postal initialize` which i don't know how they would work in portainer.

**Comment by willpower232** [comment]
I don't know too much about running things in portainer however if you are giving portainer your docker socket, as I believe is recommended, then you can get visibility of the Postal docker containers in portainer and still run things as the instructions expect.
